docs/start/api.rpc.md

```diff
@@ -0,0 +1,70 @@
+
+Since you are already familiar with the `api.query` interface, the `api.rpc` interface follows the same format, for instance -
+
+```js
+...
+
+// retrieve the chain name
+const chain = await api.rpc.system.chain();
+
+// retrieve the latest header
+const lastHeader = await api.rpc.chain.getHeader();
+
+// log the information
+console.log(`${chain}: last block #${lastHeader.number} has hash ${lastHeader.hash}`);
+```
+
+In this example, you will see the same pattern as with queries: each result is a promise and a simple `await` makes the query and resolves with the result.
+
+## Subscriptions
+
+The RPCs lend themselves to using subscriptions, for instance in the above case you would assume that once connected, the chain won't change, however new blocks will come in at intervals and we probably want to keep track of those. We can adapt the previous example to start using subscriptions -
+
+```js
+...
+
+// subscribe to the new headers
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+  console.log(`${chain}: last block #${lastHeader.number} has hash ${lastHeader.hash}`);
+});
+```
+
+Since we are dealing with a subscription, we now pass a callback into the `subscribeNewHeads` function, and this will  be triggered on each header, as they are imported. the same pattern would apply to each of the `api.rpc.subscribe*` functions, as a last parameter, a callback is to be provided that streams the latest data.
+
+In general, whenever we create a subscription, we would like to cleanup after ourselves and unsubscribe, so assuming we only want to log the first 10 headers, the above example can be adjusted in the following manner -
+
+```js
+...
+let count = 0;
+
+// subscribe to the new headers
+const unsubHeads = await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+  console.log(`${chain}: last block #${lastHeader.number} has hash ${lastHeader.hash}`);
+
+  if (++count === 10) {
+    unsubHeads();
+  }
+});
+```
+
+Unlike single-shot queries, the value we are using `await` for on subscriptions is a function, taking no parameters (that also returns nothing) that can be used to unsubscribe for the  subscription. So in the above example we set `unsubHeads` and then call it when we wish to cancel the subscription.
+
+## Detour into derives
+
+The `api.derive` interfaces will be covered in a follow-up section, but since the above example deals with new head subscriptions, a quick detour is warranted. The derives are just helpers that define certain functions and combine results from multiple sources. For new headers, the following information is useful in certain scenarios -
+
+```js
+...
+const unsub = await api.derive.chain.subscribeNewHeads((lastHeader) => {
+  console.log(`#${lastHeader.number} was authored by ${lastHeader.author}`);
+});
+```
```

Comment:
I'm getting `#108 was authored by undefined`

Reply:
Ahh, that's because I was using the node template which doesn't have `api.query.session`